views/templates/viewResult.tpl

```diff
@@ -134,7 +134,13 @@ use oat\taoOutcomeUi\model\ResultsService;
                             <td class="dataResult" colspan="2">
                         <?php
                         if ($variable->getBaseType() === "file" && $variable->getCandidateResponse() !== '') {
-                            echo '<button id="fileDownload" class="download btn-info small" value="'.htmlspecialchars($observation["uri"]).'"><span class="icon-download"></span> '.__('Download file').'</button>';
+                            echo sprintf(
```

Comment:
Not sure that is a good idea because in that particular case using the if / else block and <?php will be close before and opened after,  which create not a good view to understand the whole logic construction

Reply:
Ok.